src/tensors/tensor.jl

```diff
@@ -466,6 +546,21 @@ blocks(t::TensorMap) = t.data
 fusiontrees(t::TrivialTensorMap) = ((nothing, nothing),)
 fusiontrees(t::TensorMap) = TensorKeyIterator(t.rowr, t.colr)
 
+"""
+    Base.getindex(t::TensorMap{<:IndexSpace,N₁,N₂,I},
+                  sectors::Tuple{Vararg{I}}) where {N₁,N₂,I<:Sector} 
```

Comment:
```suggestion
                  sectors::NTuple{N₁ + N₂, I}) where {N₁,N₂,I<:Sector} 
```
This pattern does not work for julia syntax, but maybe we can just use it in the docstring?

Reply:
Certainly fine by me, looks more clear, but I don't really have the means to assess whether this might be bad practice...